### ISSUE TITLE
MKS H43 v1.1 on Skr 1.4 turbo

### ISSUE THREAD

**Issue by andyrok** [makerbase-mks/MKS-H43#30]
I don’t understand to connect the MKS H43 with the  Skr 1.4 Turbo…
Can anyone help me?
Thanks

**Comment by mlee12382** [comment]
On my skr mini I just used the 10pin cable to exp1 and adapter B for the H43. Then I assigned the LCD Serial Port in configuration_adv.h to 2 I think it was, I can check when I get home.

**Comment by andyrok** [comment]
I can’t connect the H 43 to the SKR 1.4 Turbo. The suggested connection is between the AUX-1 port of the Tft card and the WiFi port of the Skr... I don't have a cable that can get into the Skr socket... I tried to connect them with single cables but the screen at most lights up with the Makerbase logo and stops...🤬

**Comment by andyrok** [comment]
After a million tests nothing happens.. the screen remains on the initial logo... do you have any solution?
 the board is SKR 1.4Turbo whit TMC2029

**Comment by mlee12382** [comment]
Did you define an lcd serial port in configuration_adv.h under the dgus section? Ie. #if HAS_DGUS_LCD
  #define LCD_SERIAL_PORT 6
  #define LCD_BAUDRATE 115200

**Comment by andyrok** [comment]
> Did you define an lcd serial port in configuration_adv.h under the dgus section? Ie. #if HAS_DGUS_LCD
> #define LCD_SERIAL_PORT 6
> #define LCD_BAUDRATE 115200

Are you sure about 6..?
I try 👍

**Comment by mlee12382** [comment]
> > Did you define an lcd serial port in configuration_adv.h under the dgus section? Ie. #if HAS_DGUS_LCD
> > #define LCD_SERIAL_PORT 6
> > #define LCD_BAUDRATE 115200
> 
> Are you sure about 6..?
> I try 👍

I don't think it matters but that's what it is on mine. I think it just needs to be different than something you're using elsewhere.

**Comment by mlee12382** [comment]
![Screenshot_20220915-134133_GitHub.jpg](https://user-images.githubusercontent.com/89716126/190504665-27bae0ea-a1f9-4d45-a8f0-956e74af3450.jpg)

**Comment by andyrok** [comment]
I’ve try this option( LCD_SERIAL…) but not other number then 0…. I try tomorrow..👍
